examples/getplugins/main.go

```diff
@@ -18,13 +18,13 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// get a list of Modules
-	modules, err := client.UserFunctions("userID", "namespace")
+	// get a list of Plugins
```

Comment:
```suggestion
	// get a list of plugins
```
Nit based on feedback in other repos.